GDJS/Runtime/Cordova/www/index.html

```diff
@@ -22,6 +22,10 @@
 				//Initialization
 				var game = new gdjs.RuntimeGame(gdjs.projectData, {}/*GDJS_ADDITIONAL_SPEC*/);
 
+				// Set propper screen orientation
```

Comment:
Nice! Would you be able to move this part in the RuntimeGame/RuntimeGame renderer?

The index.html code should be as minimal as possible - there is almost already too much in it that could have been handled by the runtime game renderer.

Reply:
also typo: *proper

Reply:
It is cordova only, so I thought that would be the right place?

Reply:
While it's true that this file is only for Cordova, it's still an "entry point" and an entry point should not contain a lot of logic usually (we can't put every cordova specific stuff in this file).
Instead, let's think about the feature: what are we even doing? We're doing something that set up the orientation of the game on the screen. Fair enough! What is this related to in the game engine? A scene, an object, a game? Game seems the most reasonable answer (it's something related to the "game window" in a way, it's almost a global setting as it's in the project properties). It's also not specific to mobile device when we think about it! We could imagine in the future that on desktop (using Electron) or on the web, we could set the orientation (even if it's unlikely to be the case). So let's add a `_setupOrientation` in the runtime game or the runtime game renderer, that we call automatically at the beginning (advantage: if later we add a feature to change this during the game, we can refactor/reuse this method!).
So there is one remaining question: should the code dealing with Cordova be in the runtime game or the runtime game renderer?

Here, you can think of: is what I'm doing applicable to another renderer than Pixi.js? The answer is not really, because a renderer like Cocos2d-JS could run in something that is not a browser - in which case you'll need to use this framework own method to set up the orientation. So let's put this in the Runtime Game renderer! :) 
If later we implement another renderer, we'll adapt this `_setupOrientation` method to the renderer (or just share it/copy paste it if the renderer is also running in a browser environment).